apis/discoveryengine/v1alpha1/engine_reference.go

```diff
@@ -61,10 +67,10 @@ func (r *DiscoveryEngineEngineRef) NormalizedExternal(ctx context.Context, reade
 	}
 
 	// From the Config Connector object
-	if r.Namespace == "" {
-		r.Namespace = otherNamespace
-	}
 	key := types.NamespacedName{Name: r.Name, Namespace: r.Namespace}
+	if key.Namespace == "" {
+		key.Namespace = defaultNamespace
+	}
```

Comment:
oh no, I thought this has been fixed in the template.

Reply:
Probably, but we're only at December